avidemux/qt4/ADM_userInterfaces/ADM_filters/Q_seekablePreview.h

```diff
@@ -34,7 +34,7 @@ class Ui_seekablePreviewWindow : public QDialog
 	Ui_seekablePreviewDialog ui;
         
 public:        
-                Ui_seekablePreviewWindow(QWidget *parent, ADM_coreVideoFilter *videoStream, uint32_t defaultFrame = 0);
+                Ui_seekablePreviewWindow(QWidget *parent, ADM_coreVideoFilter *videoStream, QString const& title=QString(""), uint32_t defaultFrame = 0);
```

Comment:
I wonder why not just calling `previewDialog->setWindowTitle(title)` from `filtermainWindow::preview`?

Reply:
indeed